package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "reactjs-mentoring",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "start:dev": "webpack serve --config webpack.dev.config.js --open --progress",
+    "start:prod": "webpack serve --config webpack.prod.config.js --open",
+    "build:prod": "webpack --config webpack.prod.config.js",
+    "build:dev": "webpack --config webpack.dev.config.js",
+    "code:check": "./node_modules/.bin/eslint '**/*.js'",
+    "code:fix": "./node_modules/.bin/eslint '**/*.js' --fix",
+    "prepare": "husky install"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/katerinastolyarova/reactJS-mentoring.git"
+  },
+  "author": "Katerina Stolyarova",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/katerinastolyarova/reactJS-mentoring/issues"
+  },
+  "homepage": "https://github.com/katerinastolyarova/reactJS-mentoring#readme",
+  "dependencies": {
+    "clean-webpack-plugin": "^4.0.0",
+    "cross-env": "^7.0.3",
+    "html-webpack-plugin": "^5.5.0",
+    "jest": "^27.3.1",
+    "react": "^17.0.2",
+    "webpack": "^5.64.1"
```

Comment:
Please, clean your dependencies, I believe the only packages that should stay are react and cross-env
everything else is devDependency

Reply:
Fixed